docs/university.md

```diff
@@ -1005,6 +1005,18 @@ xskb1 对应 <http://www.auto.uestc.edu.cn/index/xskb1.htm>
 
 <Route author="Jiangming1399" example="/gdut/news" path="/gdut/news"/>
 
+### 通知公文网
+
+<Route author="Jim Kirisame" example="/gdut/oa_news" path="/gdut/oa_news/:category" :paramsDesc="['分类名']">
+
+| 招标公告          | 招标结果          | 校内简讯 | 校内通知   | 公示公告         |
+| ------------- | ------------- | ---- | ------ | ------------ |
+| tender_invite | tender_result | news | notice | announcement |
+
+</Route>
+
+注：校外访问仅可获取招标公告与招标结果，其他类别需要在校内网络自建才可获取。
```

Comment:
Looks like all of the categories can be visited from networks outside the campus.